.github/workflows/ci.yaml

```diff
@@ -28,16 +28,12 @@ jobs:
         run: |
           rustup target add wasm32-unknown-unknown
           cargo install cargo-make
-          cargo install nextest
 
       - name: Lint & Clippy
         run: cargo make lint
 
-      - name: Build (Release)
+      - name: Build
         run: cargo make build
 
-      - name: Run Tests
+      - name: Tests
         run: cargo make test
-
-      - name: Run Nextest
-        run: cargo make nextest
```

Comment:
No need to run tests twice (i.e. This isn't really necessary since its the same as the tests above).

Reply:
Nextest should probably be preferred since it is faster

Reply:
I can switch it if you like but it looks like it's too late.

Reply:
I got it, it's all good.